jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -654,4 +661,38 @@ Args:
     (send *ri* :go-pos-unsafe 0 0 -80) ;; face the front against the trash can
     success-move-to-trashcan-front))
 
+(defun padding-byte-array (byte-array target-length)
+  (concatenate cons byte-array (make-list (- target-length (length byte-array)) :initial-element 0)))
```

Comment:
Perhaps this function can be added to euslisp default code.(https://github.com/euslisp/euslisp or jskeus?)

Reply:
Thank, I will add it if there is appropreate library.